docs/modules/release-notes/pages/0.26.adoc

```diff
@@ -19,51 +21,484 @@ To get started, follow xref:pkl-cli:index.adoc#installation[Installation].#
+* Optimization: `const` access is checked when variables are resolved, instead of every time the variable is accessed (https://github.com/apple/pkl/pull/438[#438]).
+* Optimization: use logical AND instead of bitwise AND when comparing numbers (https://github.com/apple/pkl/pull/102[#102])
+* Mark Java classes `final` (https://github.com/apple/pkl/pull/458[#458]).
+* Migrate code to newer JDK17 features, and clean up existing Java code (https://github.com/apple/pkl/pull/451[#451], https://github.com/apple/pkl/pull/458[#458], https://github.com/apple/pkl/pull/512[#512]).
+* Improve rendering of string values within generated `<file>.pkl-expected.pcf` files when running `pkl test` (https://github.com/apple/pkl/pull/416[#416]).
+* Improve performance of loading Pkl's built-in CA certificates (https://github.com/apple/pkl/pull/518[#518]) 
+
+== Bug Fixes [small]#🐜#
+
+The following bugs have been fixed.
+
+* `Collection#sortWith` produces un-sorted output (https://github.com/apple/pkl/pull/394[#394]).
+* Property `typedType` in module `pkl.reflect` reflects upon `Type`, instead of `Typed` (https://github.com/apple/pkl/pull/426[#426]).
+* `const` members can be assigned to when via an object spread  (https://github.com/apple/pkl/pull/428[#428]).
+* Relative globbed reads resolve to the same value in different modules (https://github.com/apple/pkl/pull/449[#449]).
+* Performance bug: globbed imports and globbed reads expand to an unbounded number of root nodes (https://github.com/apple/pkl/pull/449[#449]).
+* Relative globbed imports within a package match no modules (https://github.com/apple/pkl/pull/496[#496]).
+* Constraints within typealiases resolve to the wrong values (https://github.com/apple/pkl/pull/144[#144]).
+* Members of `pkl.reflect` cannot be rendered (https://github.com/apple/pkl/pull/170[#170], https://github.com/apple/pkl/pull/470[#470]).
+* Throws `NullPointerException` if "List Resources Response" or "List Modules Response" messages contain both null `pathElements` and `error` (https://github.com/apple/pkl/pull/480[#480]).
+* Classes of `com.oracle.truffle` are not shaded in pkl-config-java-all and pkl-tools (https://github.com/apple/pkl/pull/238[#238]).
+* Throws `PklBugException` when running publish check on an invalid URL (https://github.com/apple/pkl/pull/441[#441]).
+
+== Contributors [small]#🙏#
+
+We would like to thank the contributors to this release (in alphabetical order):
```

Comment:
Looks like I'm missing here since @bioball landed the final PR for scheme agnostic projects

Reply:
Ah, apologies! Will make sure to add you.

Reply:
Added